src/SwaggerUiServiceProvider.php

```diff
@@ -36,12 +37,13 @@ public function register() : void
 
     protected function loadRoutes() : void
     {
-        Route::middleware(config('swagger-ui.middleware', ['web', EnsureUserIsAuthorized::class]))
-            ->prefix(config('swagger-ui.path'))
-            ->group(function () {
-                Route::view('/', 'swagger-ui::index')->name('swagger-ui');
-
-                Route::get('openapi.json', OpenApiJsonController::class)->name('swagger-openapi-json');
-            });
+        collect(config('swagger-ui.files'))->each(function ($values) {
+            Route::middleware($values['middleware'] ?? ['web', EnsureUserIsAuthorized::class])
+                ->group(function () use ($values) {
+                    Route::get(ltrim($values['path'], '/'), SwaggerViewController::class);
```

Comment:
```suggestion
                    Route::get($values['path'], SwaggerViewController::class);
```
trim is not needed here I think